src/index.js

```diff
@@ -81,7 +81,7 @@ export const sizeSnapshot = (options?: Options = {}): Plugin => {
       // brodybits/rollup-plugin-size-snapshot#17
       // but a reproduction is needed to add a test case, see
       // https://github.com/brodybits/rollup-plugin-size-snapshot/issues/19
-      const minified = minify(source).code;
+      const minified = (await minify(source)).code;
```

Comment:
OK for now, Terser error handling still looks like a loose end as I said in https://github.com/brodybits/rollup-plugin-size-snapshot/issues/17#issuecomment-1402324621